skimage/transform/histogram_matching.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+
+
+def _get_separate_channels(image):
+    """Return an array consisting of all channels of an image"""
+    if len(image.shape) < 3:
+        return np.asarray([image])
+    else:
+        return image.transpose(2, 0, 1)
+
+
+def _match_array_values(a, b):
+    """Return modified a array with interpolated values that correspond to the
+    values of the b array"""
```

Comment:
Using `source` instead of `a` and `template` instead of `b` (like in the original example from stackoverflow) might make the code clearer and easier to read.

E.g. `a_quantiles` -> `src_quantiles`, `b_quantities` -> `tmp_quantities`

Reply:
Right, renamed the variables accordingly.